syntax = "proto3";

package hello;

service Hello {
  rpc hello(HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
  string file_name = 1;
  string file_data = 2;
}

message HelloResponse {
  string message = 1;
}